backend/services/studentService.js

```diff
@@ -28,8 +29,21 @@ module.exports = {
         }
         return studentDAO.setStudentSkills(studentId, skills)    
     },
-    // getStudentsBySkills: (skills) => {
-    //     let skillIds = skills.map(skill => skill.skill_id);
-    //     return studentDAO.getStudentsBySkills(skillIds);
-    // },
+
+    getStudentsBySkills: async (skills) => {
+        let skillIds = skills.map(skill => skill.skill_id);
+        return await studentDAO.getStudentsBySkills(skillIds);
+    },
+
+    getStudentProfile: async (id) => {
+        let profile = await studentDAO.getStudentById(id);
+        profile = profile[0];
+        let skillIds = await skillsDAO.getStudentSkillsById(profile.id);
+        let skills = [];
+        for(let i = 0; i < skillIds.length; i++) {
+            skills[i] = await skillsDAO.getSkillById(skillIds[i].skill_id);
```

Comment:
These rows can be deleted if we implement what i mentioned on top

Reply:
I changed it while fixing the issue above